csrankings-j.csv

```diff
@@ -1124,6 +1124,7 @@ Jingzheng Wu,Chinese Academy of Sciences,http://people.ucas.ac.cn/~jingzheng,ePU
 Jinho D. Choi,Emory University,http://www.mathcs.emory.edu/~choi,xdddblAAAAAJ
 Jinho Lee,Yonsei University,http://acsys.yonsei.ac.kr/jinho.html,pm3Fso0AAAAJ
 Jinhui Xu 0001,University at Buffalo,https://www.cse.buffalo.edu/~jinhui,NOSCHOLARPAGE
+Jinjun Xiong,University at Buffalo,https://cse.buffalo.edu/~jinjun,tRt1xPYAAAAJ

```

Comment:
Page redirects to IBM Watson page. https://researcher.watson.ibm.com/researcher/view.php?person=us-jinjun

Reply:
He is a new full-time TT hire at UB from IBM research. As shown in https://engineering.buffalo.edu/computer-science-engineering/people/faculty-directory.html, he is a full professor. Thanks for your time!